src/main/resources/hudson/plugins/build_timeout/BuildTimeoutWrapper/help-strategy.groovy

```diff
@@ -22,43 +22,43 @@
  * THE SOFTWARE.
  */
 
-package hudson.plugins.build_timeout;
+package hudson.plugins.build_timeout

```

Comment:
+1: Semicolons are optional in Groovy and recommend to be removed.
http://groovy-lang.org/style-guide.html#_no_semicolons